tests/unit/other.tcl

```diff
@@ -551,6 +551,11 @@ start_cluster 1 0 {tags {"other external:skip cluster slow"}} {
             fail "hash tables weren't resize."
         }
     } {} {needs:debug}
+
+    test "CLUSTER FORGET with invalid node ID" {
+         catch {r cluster forget 1} err
+         set _ $err
+    } {*ERR Unknown node*} 
```

Comment:
For future reference, you can do this in a one liner:
```
assert_error {*ERR Unknown node*} {r cluster forget 1}
```
The current pattern is useful primarily when the last command is already returning something.